GestureHandler.web.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import {
+  ScrollView,
+  Slider,
+  Switch,
+  TextInput,
+  ToolbarAndroid,
+  ViewPagerAndroid,
+  DrawerLayoutAndroid,
+  WebView,
```

Comment:
This module is being removed from React Native and isn't included in `react-native-web` either

Reply:
Which one in detail? Currently, I see that all of these components are exported though some of them as `UnimplementedView` which is fine I guess.

Reply:
https://github.com/react-native-community/discussions-and-proposals/pull/3

Reply:
Thanks for the info!